src/Components/Login/Login.js

```diff
@@ -1,70 +1,108 @@
-import React from 'react'
+import React, { useContext, useEffect } from 'react'
+import { loginUser } from '../../apiCalls'
+import { UserContext } from '../../Context/UserContext'
 import style from './Login.module.scss'
 import aboutIcon from '../../assets/navbar/aboutNavIcon.png'
 import settingsIcon from '../../assets/navbar/settingsNavIcon.png'
 import statsIcon from '../../assets/navbar/statsNavIcon.png'
 import timerIcon from '../../assets/navbar/timerNavIcon.png'
 
 const Login = () => {
+  const [user, setUser] = useContext(UserContext)
+
+  useEffect(() => {
```

Comment:
Looking back at all the functionality we added here, I'm really proud of us for figuring this all out. 
Still a little unsure why adding this render function in the useEffect works, where directly calling the onSignIn method on the SignIn button didn't, but just happy it's all working as expected!!

Reply:
I agree!!! Not quite sure what we missed with the original Google Docs, but great job finding this solution!